ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -313,19 +315,28 @@ const variantSearchLinks = (variant, mainTranscript, genesById, user) => {
 
   const linkVariant = { genes, variations, hgvsc, ...variant }
 
+  const seqrSearchLink = (elasticsearchEnabled || svType) ? (
+    <SearchResultsLink
+      buttonText="seqr"
+      genomeVersion={genomeVersion}
+      svType={svType}
+      variantId={svType ? null : variantId}
+      location={svType && (
+        (endChrom && endChrom !== chrom) ? `${chrom}:${pos - 50}-${pos + 50}` : `${chrom}:${pos}-${end}%20`)}
+    />
+  ) : (
+    <NavLink
+      to={`/summary_data/variant_lookup?variantId=${variantId}&genomeVersion=${genomeVersion}&include_genotypes=true`}
```

Comment:
I don't see `summary_data/variant_lookup` in `urls.py,`  should this just be `/variant_lookup?...`?

Reply:
So the distinction between api routes for fetching data and top-level routes for rendering the seqr app can be a bit confusing. 

This is not linking to an `/api` endpoint for fetching data, its essentially an anchor tag linking to an rendered seqr app page. urls.py includes `'summary_data/.*'` as a route under the `react_app_pages` which means this page will render the main app html with js and then the exact page routing will be handled using the react apps routing.

If you lookin the SummaryData.jsx component you can see how all the subpaths are routed, and `variant_lookup` routes to the `VariantLookup` component. In that component, you can see that it sets `'/api/variant_lookup'` as the url for data fetching: https://github.com/broadinstitute/seqr/blob/master/ui/pages/SummaryData/components/VariantLookup.jsx#L48

Reply:
riiiight I remember this, I should've been looking in `app.jsx` to where the component routes are defined.